quickhttpd/src/application.h

```diff
@@ -34,6 +34,8 @@ class Application : public QCoreApplication
 	static Application *instance();
 	AppCliOptions* cliOptions() {return m_cliOptions;}
 	qf::core::sql::Connection sqlConnetion();
+
+  QString profile();
```

Comment:
odsazeni

Reply:
Sorry, only first commit should go to pull request. I messed it up, last two were my experiments, I intended to try them in my fork only and forget to make separate branch for them.
For your information:
We plan to use quickevent, quickshow and quickhttpd at Sudety OPEN competition, held in Ostrov u Tise. There is no internet connection and we want to make Wifi island (in Ostrov :-)) for publishing of results. Competitors will check their results mainly on their mobile phones.

Reply:
Indentation fixed, but all adding --profile option to quickhttpd is only my experiment. I have very little knowledge of C++ and did it only blindly following the pattern in quickshow.
I just need to render results, based on database. I do not need startlists, I even do not need http server, I'll use nginx. Luckily I can avoid creating http server using option --http-port 0 (not mentioned in help).

Reply:
Still one issue remains: How to tell quickhttpd that I want to render pages in particular language (Czech)? It seems, that ResultExport in quickevent it does right using language setting of quickevent, but how to do it in quickhttpd?

Reply:
> Sorry, only first commit should go to pull request. I messed it up, last two were my experiments, I intended to try them in my fork only and forget to make separate branch for them.

In order to merge the pull request, please revert any non-related commits.
run `git revert {commit hash}` for each, e.g. `git revert 9b3c8fd` will revert `Fix indent` commit,
or reset the branch to include only first commit:
```
// make sure you are on html_responsive branch
git reset --hard d51cee1
git push --force origin html_responsve
```

Reply:
OK, done.

Reply:
> I just need to render results, based on database. I do not need startlists

Can you just silently ignore the start-lists generated?

Reply:
Of course I do. If it has no/minimal impact on performance.
It was just an experiment, how hard is C++ programming for me. :-) (Quite a lot.)

Reply:
try to start with #625 :)